hw-7.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+class Universe {
+    constructor() {
+        this.location='Seven Kingdoms';
+        this.creator='George R. R. Martin';
+    }
+    set = () => {
+        console.log('My homeland is ' + this.location + ' created by ' + this.creator)
+    }
+}
+
+class TheNorth extends Universe {
+    constructor(logo, leader) {
+        super();
+        this.logo = logo;
+        this.leader = leader;
+    }
+    setLeader(leader) {
+        if (['Eddard Stark', 'Ramsay Boltone'].includes(leader)) {
+            console.log(this.leader + 'is in the North of ' + this.location)
+        } else {
+            throw new Error('Leader is no in the North');
+        }
+    }
```

Comment:
this function is named `setLeader` but without a setting.
probably should be three methods:
1) validate leade
2) set leader
3) leader greetings